### DISCUSSION TITLE
Pull Request Approvals

### DISCUSSION THREAD

**Discussion by tlenzmeier58** [community/community#28362] [Other Feature Feedback, Questions, & Ideas]
I have several members on my team who are new to GitHub. Therefore, I want everyone on the team to get my approval for every pull request. I can see where I protect the branch, but am not sure how to name me as the only approver.

**Comment by jfullstackdev** [comment]
![image](https://user-images.githubusercontent.com/47092464/182890095-e406d34e-36be-4563-a085-5e89d3e595e3.png)

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews

these are the possible scenarios that you can do above,

**Reply by Shane32** [reply]
I believe you can specify that 'code owners' must approve PRs.  See branch protection rules settings:

![image](https://user-images.githubusercontent.com/6377684/182986095-68d27081-0230-4b08-bdf6-9b1fca3cbd20.png)

You can learn about code owners here:

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

The `CODEOWNERS` file can be as simple as:

```
* @tlenzmeier58 
```

Then all PRs would need your approval.  Note that this functionality is only available in public repositories or for private repositories if you have some sort of paid subscription (GitHub Pro / Teams / Enterprise / etc).
